framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/constants/OpenShiftTemplateConstants.java

```diff
@@ -17,6 +17,8 @@
 
 public class OpenShiftTemplateConstants {
 
+    public static final String DEFAULT_PASSWORD = "DEFAULT_PASSWORD";
```

Comment:
Please at least add comment about what this password is used for. Or better reflect it in the constant name. Maven? Git? Workbench? Not clear just from the constant name..